pkg/plugins/manager/pipeline/bootstrap/steps.go

```diff
@@ -153,6 +153,8 @@ func configureAppChildPlugin(cfg *config.Cfg, parent *plugins.Plugin, child *plu
 	if parent.IsCorePlugin() {
 		child.Module = path.Join("core:plugin", parent.ID, appSubPath)
 	} else {
+		// TODO
```

Comment:
Should this be here?

Reply:
What do you mean? What does not work without this? Because to me it seems like things worked without having to uncomment this.